package.json

```diff
@@ -23,7 +23,8 @@
     "bootstrap": "yarn install && node_modules/.bin/lerna bootstrap",
     "publish": "node_modules/.bin/lerna publish --exact",
     "changelog": "node_modules/.bin/lerna-changelog",
-    "clean": "node_modules/.bin/lerna clean"
+    "clean": "node_modules/.bin/lerna clean",
+    "dev": "bin/ssl-check && yarn start"
```

Comment:
I don't think you've tested this...

Reply:
I added it locally, sorry just copied over and I think you're right that it's not going to work since I added it to my package.json. Really this needs to be in the generated package.json or linked else where. Sorry for the non-working PR. I will work on putting it in the right place.